src/core/store/modules/collateral/collateral.reducer.ts

```diff
@@ -92,13 +92,33 @@ const collateralReducer = createReducer(collateralInitialState, (builder) => {
 
     // State changes from non collateral actions
 
+    /* -------------------------------- getLines ------------------------------- */
```

Comment:
Arent u missing a reducer for `getLineEvents`?

Reply:
No, because the `getLinePage` reducer is adding the `escrow`, `spigot`, and `collateralEvents` objects to the Line state.

Reply:
but we might not call getLinePage we might only call getLineEvents if we already have some data from getLines

Reply:
And we cant add collateral data to lines since they are different parts of state. 
We are currently in the collateral reducer file and we are adding the collateralMap here from getLines